static/js/permits_geo_time.js

```diff
@@ -0,0 +1,126 @@
+      }
+
+      if (!this.addButtonNode) {
+        throw "No add form node. Make sure there’s a node with `data-geo-time-role=\"addForm\"`.";
+      }
+
+      if (!this.totalFormsInputNode) {
+        throw "No total forms input node. Make sure there’s a node with `data-geo-time-role=\"managementForm\"` which contains the management form.";
+      }
+
+      this._addEventListeners();
+
+      // If you add some forms and then refresh the page, the browser restores
+      // the inputs values but not the dynamic data (the added forms), which
+      // means the TOTAL_FORMS input will be out of sync
+      this._fixTotalForms();
+    }
+
+    addForm() {
+      let newNode = this.emptyFormNode.children[0].cloneNode(true);
+      let newNodeId = parseInt(this.totalFormsInputNode.value);
+      newNode.innerHTML = newNode.innerHTML.replace(/__prefix__/g, newNodeId);
+
+      // Flag the widget to be initialized by the geometry widget manager
+      newNode.querySelector("[data-geometry-widget]").dataset.initialize = "1";
+      // Set the correct form number to appear in the form title
+      newNode.querySelector("[data-geo-time-role='formNumber']").textContent = newNodeId + 1;
+
+      // Manually set bindings for the datepickers
+      // See https://github.com/monim67/django-bootstrap-datepicker-plus/blob/22c4299019cb6328eed2938598e323c0d43c5e9a/bootstrap_datepicker_plus/static/bootstrap_datepicker_plus/js/datepicker-widget.js
+      newNode.querySelectorAll("[dp_config]").forEach(function(node) {
+        let $picker = jQuery(node).datetimepicker(JSON.parse(node.attributes.dp_config.value).options);
+
+        $picker.next('.input-group-addon').on('click', function(){
+          $picker.data("DateTimePicker").show();
+        });
+      });
+      this._copyFormData(
+        [...this.formsContainerNode.querySelectorAll("[data-geo-time-role='form']")].pop(),
+        newNode,
+        newNodeId
+      );
+
+      this.formsContainerNode.appendChild(newNode);
+
+      [...this.formsContainerNode.querySelectorAll(".collapse")].forEach(node => {
+        jQuery(node).collapse("hide");
+      });
+
+      let collapseNode = newNode.querySelector(".collapse");
+      this._fixMapSizeOnCollapseOpen(collapseNode);
+      jQuery(collapseNode).collapse("show");
+
+      // Initialize `geometryWidget` on the newly created element
+      window.geometryWidgetManager.rebind();
+
+      this.totalFormsInputNode.value++;
+    }
+
+    _addEventListeners() {
+      this.addButtonNode.addEventListener("click", this.addForm.bind(this));
+
+      [...this.formsContainerNode.querySelectorAll(".collapse")].forEach(
+        this._fixMapSizeOnCollapseOpen
+      );
+    }
+
+    _fixMapSizeOnCollapseOpen(collapseNode) {
+      jQuery(collapseNode).on("shown.bs.collapse", (e) => {
+        let geometryWidgetNode = collapseNode.querySelector("[data-geometry-widget]");
```

Comment:
Throw an error in the console here. Selector return null

Reply:
Need to check once the issu is fixed if we can add manually coordinate because for now it's seem to be broken